include/gmapping/gridfastslam/gridslamprocessor.h

```diff
@@ -305,11 +308,10 @@ namespace GMapping {
     // stream in which to write the messages
     std::ostream& m_infoStream;
     
-    
     // the functions below performs side effect on the internal structure,
     //should be called only inside the processScan method
   private:
-    
+    std::vector<double> durations;
```

Comment:
why is that a member and not created before the for loop in the scanMatch function ?